Lombiq.HelpfulLibraries.RestEase/Lombiq.HelpfulLibraries.RestEase.csproj

```diff
@@ -0,0 +1,57 @@
+<Project Sdk="Microsoft.NET.Sdk.Razor">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <DefaultItemExcludes>$(DefaultItemExcludes);.git*</DefaultItemExcludes>
+    <RootNamespace>Lombiq.HelpfulLibraries.RestEase</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Remove="Libraries\Contents\**" />

```

Comment:
Ez az egész blokk miért szükséges? Nekem lefordul nélküle is.

Reply:
Nem szükséges